  INFO root [0x00002f00] [2018-06-21 09:33:04,482] - 
  INFO root [0x00002f00] [2018-06-21 09:33:04,492] - -----------------------------------------------------------------------------------------


  INFO root [0x00002f00] [2018-06-21 09:33:04,492] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002f00] [2018-06-21 09:33:04,492] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002f00] [2018-06-21 09:33:04,492] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002f00] [2018-06-21 09:33:04,532] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2617 )
  INFO root [0x00002f00] [2018-06-21 09:33:04,532] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2617, SOMAXCONN=5.
  INFO root [0x00002f00] [2018-06-21 09:33:04,533] - server-configs:
  INFO root [0x00002f00] [2018-06-21 09:33:04,533] - 	gameUpdateHertz : 10
  INFO root [0x00002f00] [2018-06-21 09:33:04,533] - 	defaultAoIRadius : 80
  INFO root [0x00002f00] [2018-06-21 09:33:04,534] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00002f00] [2018-06-21 09:33:04,534] - 	entryScriptFile : kbemain
  INFO root [0x00002f00] [2018-06-21 09:33:04,534] - 	internalAddr : 192.168.31.56:2617
  INFO root [0x00002f00] [2018-06-21 09:33:04,534] - 	componentID : 8001
  INFO root [0x00002f00] [2018-06-21 09:33:06,044] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 23460. PID: 11884 ----
  INFO root [0x00002f00] [2018-06-21 09:33:06,044] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002f00] [2018-06-21 09:33:06,044] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002f00] [2018-06-21 09:33:06,064] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002f00] [2018-06-21 09:33:06,214] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002f00] [2018-06-21 09:33:06,214] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002f00] [2018-06-21 09:33:06,214] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002f00] [2018-06-21 09:33:06,224] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002f00] [2018-06-21 09:33:06,224] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002f00] [2018-06-21 09:33:06,224] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002f00] [2018-06-21 09:33:06,224] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002f00] [2018-06-21 09:33:06,274] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002f00] [2018-06-21 09:33:06,324] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002f00] [2018-06-21 09:33:06,324] - Script::install(): KBExtra is successfully!
  INFO root [0x00002f00] [2018-06-21 09:33:06,344] - TelnetServer server is running on port 50000
  INFO root [0x00002f00] [2018-06-21 09:33:07,768] - ---- cellapp is running ----
 DEBUG root [0x00002f00] [2018-06-21 09:33:07,769] - Components::process(): Request for the process of identity...
  WARN root [0x00002f00] [2018-06-21 09:33:11,175] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20460, errorno=10048
  WARN root [0x00002f00] [2018-06-21 09:33:14,205] - EntityApp::checkTickPeriod: tick took 6439% (6.44 seconds)!
  INFO root [0x00002f00] [2018-06-21 09:33:15,226] - Components::findComponents: find logger(1)...
  INFO root [0x00002f00] [2018-06-21 09:33:15,276] - Components::findComponents: found logger, addr:192.168.31.56:2624
  INFO root [0x00002f00] [2018-06-21 09:33:15,276] - Components::addComponent: reset orderLog, uid=23460!
  INFO root [0x00002f00] [2018-06-21 09:33:15,276] - Components::addComponent[logger], uid=23460, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002f00] [2018-06-21 09:33:15,276] - Components::findComponents: find dbmgr(2)...
  WARN root [0x00002f00] [2018-06-21 09:33:16,376] - Forwarding logs to logger[192.168.31.56:2624/0/0/0]...
 ERROR root [0x00002f00] [2018-06-21 09:33:33,819] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002f00] [2018-06-21 09:33:33,824] - The following logs sent to logger failed:
  WARN root [0x00002f00] [2018-06-21 09:33:33,824] - ==> [2018-06-21 09:33:33 818] TCPPacketReceiver::processPendingEvents(192.168.31.56:2624): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002f00] [2018-06-21 09:33:33,825] - ==> [2018-06-21 09:33:33 820] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002f00] [2018-06-21 09:33:34,220] - Components::findComponents: find dbmgr(11)...
  INFO root [0x00002d9c] [2018-06-21 09:34:04,505] - 
  INFO root [0x00002d9c] [2018-06-21 09:34:04,507] - -----------------------------------------------------------------------------------------


  INFO root [0x00002d9c] [2018-06-21 09:34:04,509] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002d9c] [2018-06-21 09:34:04,510] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002d9c] [2018-06-21 09:34:04,511] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002d9c] [2018-06-21 09:34:04,524] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2658 )
  INFO root [0x00002d9c] [2018-06-21 09:34:04,525] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2658, SOMAXCONN=5.
  INFO root [0x00002d9c] [2018-06-21 09:34:04,525] - server-configs:
  INFO root [0x00002d9c] [2018-06-21 09:34:04,526] - 	gameUpdateHertz : 10
  INFO root [0x00002d9c] [2018-06-21 09:34:04,526] - 	defaultAoIRadius : 80
  INFO root [0x00002d9c] [2018-06-21 09:34:04,526] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00002d9c] [2018-06-21 09:34:04,527] - 	entryScriptFile : kbemain
  INFO root [0x00002d9c] [2018-06-21 09:34:04,527] - 	internalAddr : 192.168.31.56:2658
  INFO root [0x00002d9c] [2018-06-21 09:34:04,527] - 	componentID : 8001
  INFO root [0x00002d9c] [2018-06-21 09:34:06,032] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 23656. PID: 5060 ----
  INFO root [0x00002d9c] [2018-06-21 09:34:06,032] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002d9c] [2018-06-21 09:34:06,042] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002d9c] [2018-06-21 09:34:06,042] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002d9c] [2018-06-21 09:34:06,271] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002d9c] [2018-06-21 09:34:06,272] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002d9c] [2018-06-21 09:34:06,273] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002d9c] [2018-06-21 09:34:06,281] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002d9c] [2018-06-21 09:34:06,282] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002d9c] [2018-06-21 09:34:06,282] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002d9c] [2018-06-21 09:34:06,283] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002d9c] [2018-06-21 09:34:06,465] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002d9c] [2018-06-21 09:34:06,615] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002d9c] [2018-06-21 09:34:06,616] - Script::install(): KBExtra is successfully!
  INFO root [0x00002d9c] [2018-06-21 09:34:06,673] - TelnetServer server is running on port 50000
  INFO root [0x00002d9c] [2018-06-21 09:34:08,026] - ---- cellapp is running ----
 DEBUG root [0x00002d9c] [2018-06-21 09:34:08,026] - Components::process(): Request for the process of identity...
  WARN root [0x00002d9c] [2018-06-21 09:34:14,107] - EntityApp::checkTickPeriod: tick took 6077% (6.08 seconds)!
 DEBUG root [0x00002d9c] [2018-06-21 09:34:14,637] - Cellapp::lookApp: 192.168.31.56:2676/0/0/0
  INFO root [0x00002d9c] [2018-06-21 09:34:15,137] - Components::findComponents: find logger(1)...
  INFO root [0x00002d9c] [2018-06-21 09:34:15,157] - Components::findComponents: found logger, addr:192.168.31.56:2656
  INFO root [0x00002d9c] [2018-06-21 09:34:15,157] - Components::addComponent: reset orderLog, uid=23656!
  INFO root [0x00002d9c] [2018-06-21 09:34:15,157] - Components::addComponent[logger], uid=23656, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002d9c] [2018-06-21 09:34:15,167] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002d9c] [2018-06-21 09:34:15,267] - Components::findComponents: found dbmgr, addr:192.168.31.56:2660
  INFO root [0x00002d9c] [2018-06-21 09:34:15,267] - Components::addComponent: reset orderLog, uid=23656!
  INFO root [0x00002d9c] [2018-06-21 09:34:15,267] - Components::addComponent[dbmgr], uid=23656, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002d9c] [2018-06-21 09:34:15,267] - Components::findComponents: found dbmgr, addr:192.168.31.56:2660
  WARN root [0x00002d9c] [2018-06-21 09:34:16,267] - EntityApp::checkTickPeriod: tick took 1230% (1.23 seconds)!
  WARN root [0x00002d9c] [2018-06-21 09:34:16,267] - Forwarding logs to logger[192.168.31.56:2656/0/0/0]...
 ERROR root [0x00002d9c] [2018-06-21 09:40:44,053] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002d9c] [2018-06-21 09:40:44,067] - The following logs sent to logger failed:
  WARN root [0x00002d9c] [2018-06-21 09:40:44,067] - ==> [2018-06-21 09:40:44 049] TCPPacketReceiver::processPendingEvents(192.168.31.56:2656): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002d9c] [2018-06-21 09:40:44,068] - ==> [2018-06-21 09:40:44 054] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002d9c] [2018-06-21 09:40:44,154] - TCPPacketReceiver::processPendingEvents(192.168.31.56:2660): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002d9c] [2018-06-21 09:40:44,160] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002d9c] [2018-06-21 09:40:44,162] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00002d9c] [2018-06-21 09:40:44,164] - Shutdowner::onShutdown: shutting down
  INFO root [0x00002d9c] [2018-06-21 09:40:44,165] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,165] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,166] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,200] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,200] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,200] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,201] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,201] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,201] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,201] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,202] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002d9c] [2018-06-21 09:40:44,222] - Script::uninstall(): is successfully!
 DEBUG root [0x00002d9c] [2018-06-21 09:40:44,223] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00002d9c] [2018-06-21 09:40:44,523] - ThreadPool::destroy(): waiting for thread(2)[0x4031b8,0x403150,], try=1
  INFO root [0x00001758] [2018-06-21 09:40:51,874] - 
  INFO root [0x00001758] [2018-06-21 09:40:51,884] - -----------------------------------------------------------------------------------------


  INFO root [0x00001758] [2018-06-21 09:40:51,886] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001758] [2018-06-21 09:40:51,886] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001758] [2018-06-21 09:40:51,886] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001758] [2018-06-21 09:40:51,917] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2752 )
  INFO root [0x00001758] [2018-06-21 09:40:51,917] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2752, SOMAXCONN=5.
  INFO root [0x00001758] [2018-06-21 09:40:51,917] - server-configs:
  INFO root [0x00001758] [2018-06-21 09:40:51,918] - 	gameUpdateHertz : 10
  INFO root [0x00001758] [2018-06-21 09:40:51,918] - 	defaultAoIRadius : 80
  INFO root [0x00001758] [2018-06-21 09:40:51,918] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00001758] [2018-06-21 09:40:51,918] - 	entryScriptFile : kbemain
  INFO root [0x00001758] [2018-06-21 09:40:51,918] - 	internalAddr : 192.168.31.56:2752
  INFO root [0x00001758] [2018-06-21 09:40:51,918] - 	componentID : 8001
  INFO root [0x00001758] [2018-06-21 09:40:53,432] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 24988. PID: 11900 ----
  INFO root [0x00001758] [2018-06-21 09:40:53,434] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001758] [2018-06-21 09:40:53,435] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001758] [2018-06-21 09:40:53,436] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001758] [2018-06-21 09:40:53,629] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001758] [2018-06-21 09:40:53,630] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001758] [2018-06-21 09:40:53,631] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001758] [2018-06-21 09:40:53,638] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001758] [2018-06-21 09:40:53,639] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001758] [2018-06-21 09:40:53,639] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001758] [2018-06-21 09:40:53,641] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001758] [2018-06-21 09:40:53,728] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001758] [2018-06-21 09:40:53,815] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001758] [2018-06-21 09:40:53,816] - Script::install(): KBExtra is successfully!
  INFO root [0x00001758] [2018-06-21 09:40:53,840] - TelnetServer server is running on port 50000
  INFO root [0x00001758] [2018-06-21 09:40:55,257] - ---- cellapp is running ----
 DEBUG root [0x00001758] [2018-06-21 09:40:55,258] - Components::process(): Request for the process of identity...
  WARN root [0x00001758] [2018-06-21 09:40:58,659] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20698, errorno=10048
  WARN root [0x00001758] [2018-06-21 09:41:01,701] - EntityApp::checkTickPeriod: tick took 6446% (6.45 seconds)!
  INFO root [0x00001758] [2018-06-21 09:41:02,761] - Components::findComponents: find logger(1)...
  INFO root [0x00001758] [2018-06-21 09:41:02,851] - Components::findComponents: found logger, addr:192.168.31.56:2756
  INFO root [0x00001758] [2018-06-21 09:41:02,851] - Components::addComponent: reset orderLog, uid=24988!
  INFO root [0x00001758] [2018-06-21 09:41:02,851] - Components::addComponent[logger], uid=24988, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001758] [2018-06-21 09:41:02,851] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001758] [2018-06-21 09:41:02,951] - Components::findComponents: found dbmgr, addr:192.168.31.56:2754
  INFO root [0x00001758] [2018-06-21 09:41:02,951] - Components::addComponent: reset orderLog, uid=24988!
  INFO root [0x00001758] [2018-06-21 09:41:02,951] - Components::addComponent[dbmgr], uid=24988, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001758] [2018-06-21 09:41:02,951] - Components::findComponents: found dbmgr, addr:192.168.31.56:2754
  WARN root [0x00001758] [2018-06-21 09:41:03,951] - EntityApp::checkTickPeriod: tick took 1289% (1.29 seconds)!
  WARN root [0x00001758] [2018-06-21 09:41:03,951] - Forwarding logs to logger[192.168.31.56:2756/0/0/0]...
 ERROR root [0x00001758] [2018-06-21 22:33:34,591] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002510] [2018-06-21 22:33:43,175] - 
  INFO root [0x00002510] [2018-06-21 22:33:43,177] - -----------------------------------------------------------------------------------------


  INFO root [0x00002510] [2018-06-21 22:33:43,180] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002510] [2018-06-21 22:33:43,180] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002510] [2018-06-21 22:33:43,181] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002510] [2018-06-21 22:33:43,201] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10049 )
  INFO root [0x00002510] [2018-06-21 22:33:43,201] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10049, SOMAXCONN=5.
  INFO root [0x00002510] [2018-06-21 22:33:43,202] - server-configs:
  INFO root [0x00002510] [2018-06-21 22:33:43,202] - 	gameUpdateHertz : 10
  INFO root [0x00002510] [2018-06-21 22:33:43,202] - 	defaultAoIRadius : 80
  INFO root [0x00002510] [2018-06-21 22:33:43,202] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00002510] [2018-06-21 22:33:43,202] - 	entryScriptFile : kbemain
  INFO root [0x00002510] [2018-06-21 22:33:43,203] - 	internalAddr : 192.168.31.56:10049
  INFO root [0x00002510] [2018-06-21 22:33:43,203] - 	componentID : 8001
  INFO root [0x00002510] [2018-06-21 22:33:44,712] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 12576. PID: 12960 ----
  INFO root [0x00002510] [2018-06-21 22:33:44,713] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002510] [2018-06-21 22:33:44,713] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002510] [2018-06-21 22:33:44,714] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002510] [2018-06-21 22:33:44,893] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002510] [2018-06-21 22:33:44,894] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002510] [2018-06-21 22:33:44,894] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002510] [2018-06-21 22:33:44,899] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002510] [2018-06-21 22:33:44,900] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002510] [2018-06-21 22:33:44,900] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002510] [2018-06-21 22:33:44,910] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002510] [2018-06-21 22:33:44,993] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002510] [2018-06-21 22:33:45,078] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002510] [2018-06-21 22:33:45,079] - Script::install(): KBExtra is successfully!
  INFO root [0x00002510] [2018-06-21 22:33:45,143] - TelnetServer server is running on port 50000
  INFO root [0x00002510] [2018-06-21 22:33:46,798] - ---- cellapp is running ----
 DEBUG root [0x00002510] [2018-06-21 22:33:46,799] - Components::process(): Request for the process of identity...
  WARN root [0x00002510] [2018-06-21 22:33:50,176] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20862, errorno=10048
  WARN root [0x00002510] [2018-06-21 22:33:53,216] - EntityApp::checkTickPeriod: tick took 6416% (6.42 seconds)!
  INFO root [0x00002510] [2018-06-21 22:33:54,300] - Components::findComponents: find logger(1)...
  INFO root [0x00002510] [2018-06-21 22:33:54,314] - Components::findComponents: found logger, addr:192.168.31.56:10047
  INFO root [0x00002510] [2018-06-21 22:33:54,315] - Components::addComponent: reset orderLog, uid=12576!
  INFO root [0x00002510] [2018-06-21 22:33:54,315] - Components::addComponent[logger], uid=12576, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002510] [2018-06-21 22:33:54,317] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002510] [2018-06-21 22:33:54,419] - Components::findComponents: found dbmgr, addr:192.168.31.56:10045
  INFO root [0x00002510] [2018-06-21 22:33:54,419] - Components::addComponent: reset orderLog, uid=12576!
  INFO root [0x00002510] [2018-06-21 22:33:54,420] - Components::addComponent[dbmgr], uid=12576, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002510] [2018-06-21 22:33:54,421] - Components::findComponents: found dbmgr, addr:192.168.31.56:10045
  WARN root [0x00002510] [2018-06-21 22:33:55,421] - EntityApp::checkTickPeriod: tick took 1221% (1.22 seconds)!
  WARN root [0x00002510] [2018-06-21 22:33:55,422] - Forwarding logs to logger[192.168.31.56:10047/0/0/0]...
 ERROR root [0x00002510] [2018-06-21 22:33:55,529] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002510] [2018-06-21 22:33:55,531] - The following logs sent to logger failed:
  WARN root [0x00002510] [2018-06-21 22:33:55,531] - ==> [2018-06-21 22:33:55 527] TCPPacketReceiver::processPendingEvents(192.168.31.56:10047): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002510] [2018-06-21 22:33:55,532] - ==> [2018-06-21 22:33:55 529] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002808] [2018-06-21 22:34:10,684] - 
  INFO root [0x00002808] [2018-06-21 22:34:10,685] - -----------------------------------------------------------------------------------------


  INFO root [0x00002808] [2018-06-21 22:34:10,687] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002808] [2018-06-21 22:34:10,687] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002808] [2018-06-21 22:34:10,688] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002808] [2018-06-21 22:34:10,698] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10066 )
  INFO root [0x00002808] [2018-06-21 22:34:10,699] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10066, SOMAXCONN=5.
  INFO root [0x00002808] [2018-06-21 22:34:10,699] - server-configs:
  INFO root [0x00002808] [2018-06-21 22:34:10,699] - 	gameUpdateHertz : 10
  INFO root [0x00002808] [2018-06-21 22:34:10,699] - 	defaultAoIRadius : 80
  INFO root [0x00002808] [2018-06-21 22:34:10,699] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00002808] [2018-06-21 22:34:10,700] - 	entryScriptFile : kbemain
  INFO root [0x00002808] [2018-06-21 22:34:10,700] - 	internalAddr : 192.168.31.56:10066
  INFO root [0x00002808] [2018-06-21 22:34:10,700] - 	componentID : 8001
  INFO root [0x00002808] [2018-06-21 22:34:12,208] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 12668. PID: 11712 ----
  INFO root [0x00002808] [2018-06-21 22:34:12,209] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002808] [2018-06-21 22:34:12,209] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002808] [2018-06-21 22:34:12,210] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002808] [2018-06-21 22:34:12,362] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002808] [2018-06-21 22:34:12,363] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002808] [2018-06-21 22:34:12,363] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002808] [2018-06-21 22:34:12,369] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002808] [2018-06-21 22:34:12,370] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002808] [2018-06-21 22:34:12,370] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002808] [2018-06-21 22:34:12,371] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002808] [2018-06-21 22:34:12,427] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002808] [2018-06-21 22:34:12,483] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002808] [2018-06-21 22:34:12,483] - Script::install(): KBExtra is successfully!
  INFO root [0x00002808] [2018-06-21 22:34:12,509] - TelnetServer server is running on port 50000
  INFO root [0x00002808] [2018-06-21 22:34:14,085] - ---- cellapp is running ----
 DEBUG root [0x00002808] [2018-06-21 22:34:14,091] - Components::process(): Request for the process of identity...
  WARN root [0x00002808] [2018-06-21 22:34:20,303] - EntityApp::checkTickPeriod: tick took 6218% (6.22 seconds)!
  INFO root [0x00002808] [2018-06-21 22:34:21,385] - Components::findComponents: find logger(1)...
  INFO root [0x00002808] [2018-06-21 22:34:21,419] - Components::findComponents: found logger, addr:192.168.31.56:10070
  INFO root [0x00002808] [2018-06-21 22:34:21,420] - Components::addComponent: reset orderLog, uid=12668!
  INFO root [0x00002808] [2018-06-21 22:34:21,420] - Components::addComponent[logger], uid=12668, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002808] [2018-06-21 22:34:21,423] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002808] [2018-06-21 22:34:21,523] - Components::findComponents: found dbmgr, addr:192.168.31.56:10071
  INFO root [0x00002808] [2018-06-21 22:34:21,523] - Components::addComponent: reset orderLog, uid=12668!
  INFO root [0x00002808] [2018-06-21 22:34:21,523] - Components::addComponent[dbmgr], uid=12668, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002808] [2018-06-21 22:34:21,524] - Components::findComponents: found dbmgr, addr:192.168.31.56:10071
  WARN root [0x00002808] [2018-06-21 22:34:22,524] - EntityApp::checkTickPeriod: tick took 1239% (1.24 seconds)!
  WARN root [0x00002808] [2018-06-21 22:34:22,525] - Forwarding logs to logger[192.168.31.56:10070/0/0/0]...
 ERROR root [0x00002808] [2018-06-21 22:34:54,231] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002808] [2018-06-21 22:34:54,254] - The following logs sent to logger failed:
  WARN root [0x00002808] [2018-06-21 22:34:54,255] - ==> [2018-06-21 22:34:54 226] TCPPacketReceiver::processPendingEvents(192.168.31.56:10070): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002808] [2018-06-21 22:34:54,256] - ==> [2018-06-21 22:34:54 232] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002808] [2018-06-21 22:34:54,344] - TCPPacketReceiver::processPendingEvents(192.168.31.56:10071): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002808] [2018-06-21 22:34:54,348] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002808] [2018-06-21 22:34:54,365] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00002808] [2018-06-21 22:34:54,368] - Shutdowner::onShutdown: shutting down
  INFO root [0x00002808] [2018-06-21 22:34:54,369] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00002808] [2018-06-21 22:34:54,369] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00002808] [2018-06-21 22:34:54,370] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00002808] [2018-06-21 22:34:54,409] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002808] [2018-06-21 22:34:54,410] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002808] [2018-06-21 22:34:54,410] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002808] [2018-06-21 22:34:54,410] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002808] [2018-06-21 22:34:54,410] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002808] [2018-06-21 22:34:54,410] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002808] [2018-06-21 22:34:54,411] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002808] [2018-06-21 22:34:54,411] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002808] [2018-06-21 22:34:54,428] - Script::uninstall(): is successfully!
 DEBUG root [0x00002808] [2018-06-21 22:34:54,429] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00002e94] [2018-06-21 22:35:02,166] - 
  INFO root [0x00002e94] [2018-06-21 22:35:02,170] - -----------------------------------------------------------------------------------------


  INFO root [0x00002e94] [2018-06-21 22:35:02,173] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002e94] [2018-06-21 22:35:02,173] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002e94] [2018-06-21 22:35:02,174] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002e94] [2018-06-21 22:35:02,203] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10101 )
  INFO root [0x00002e94] [2018-06-21 22:35:02,203] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10101, SOMAXCONN=5.
  INFO root [0x00002e94] [2018-06-21 22:35:02,204] - server-configs:
  INFO root [0x00002e94] [2018-06-21 22:35:02,204] - 	gameUpdateHertz : 10
  INFO root [0x00002e94] [2018-06-21 22:35:02,204] - 	defaultAoIRadius : 80
  INFO root [0x00002e94] [2018-06-21 22:35:02,205] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00002e94] [2018-06-21 22:35:02,205] - 	entryScriptFile : kbemain
  INFO root [0x00002e94] [2018-06-21 22:35:02,205] - 	internalAddr : 192.168.31.56:10101
  INFO root [0x00002e94] [2018-06-21 22:35:02,206] - 	componentID : 8001
  INFO root [0x00002e94] [2018-06-21 22:35:03,717] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 12834. PID: 5752 ----
  INFO root [0x00002e94] [2018-06-21 22:35:03,718] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002e94] [2018-06-21 22:35:03,718] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002e94] [2018-06-21 22:35:03,719] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002e94] [2018-06-21 22:35:03,871] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002e94] [2018-06-21 22:35:03,872] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002e94] [2018-06-21 22:35:03,873] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002e94] [2018-06-21 22:35:03,879] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002e94] [2018-06-21 22:35:03,879] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002e94] [2018-06-21 22:35:03,880] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002e94] [2018-06-21 22:35:03,881] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002e94] [2018-06-21 22:35:03,965] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002e94] [2018-06-21 22:35:04,047] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002e94] [2018-06-21 22:35:04,048] - Script::install(): KBExtra is successfully!
  INFO root [0x00002e94] [2018-06-21 22:35:04,075] - TelnetServer server is running on port 50000
  INFO root [0x00002e94] [2018-06-21 22:35:05,811] - ---- cellapp is running ----
 DEBUG root [0x00002e94] [2018-06-21 22:35:05,812] - Components::process(): Request for the process of identity...
  WARN root [0x00002e94] [2018-06-21 22:35:11,914] - EntityApp::checkTickPeriod: tick took 6102% (6.10 seconds)!
  INFO root [0x00002e94] [2018-06-21 22:35:13,012] - Components::findComponents: find logger(1)...
  INFO root [0x00002e94] [2018-06-21 22:35:13,067] - Components::findComponents: found logger, addr:192.168.31.56:10100
  INFO root [0x00002e94] [2018-06-21 22:35:13,068] - Components::addComponent: reset orderLog, uid=12834!
  INFO root [0x00002e94] [2018-06-21 22:35:13,068] - Components::addComponent[logger], uid=12834, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002e94] [2018-06-21 22:35:13,071] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002e94] [2018-06-21 22:35:13,173] - Components::findComponents: found dbmgr, addr:192.168.31.56:10103
  INFO root [0x00002e94] [2018-06-21 22:35:13,173] - Components::addComponent: reset orderLog, uid=12834!
  INFO root [0x00002e94] [2018-06-21 22:35:13,174] - Components::addComponent[dbmgr], uid=12834, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002e94] [2018-06-21 22:35:13,174] - Components::findComponents: found dbmgr, addr:192.168.31.56:10103
  WARN root [0x00002e94] [2018-06-21 22:35:14,174] - EntityApp::checkTickPeriod: tick took 1262% (1.26 seconds)!
  WARN root [0x00002e94] [2018-06-21 22:35:14,175] - Forwarding logs to logger[192.168.31.56:10100/0/0/0]...
